client/src/modules/email-report/index.html

```diff
@@ -0,0 +1,217 @@
+
+            <!--select a profile -->
+            <div class="form-group">
+              <a  class="pull-right"
+                data-method="selectProfile"
+                ui-sref="emailreport.selectProfile"
+                ui-sref-opts="{notify:false}" translate>
+                  FORM.LABELS.SELECT_PROFILE
+              </a>
+            </div>
+
+
+            <!-- name -->
+            <div class="form-group" 
+              ng-class="{'has-error':EmailReportRegistrationForm.name.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.NAME
+              </label>
+
+              <div class="input-group col-lg-12">
+                <input class="form-control"
+                  name="name"
+                  ng-model="EmailReportCtrl.emailReport.name"
+                  autocomplete="off"
+                  required/>
+                <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.name.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html">
+                    </div>
+                </div>
+              </div>
+            </div>
+            <!-- email -->
+            <div class="form-group"
+              ng-class="{'has-error' : EmailReportRegistrationForm.email.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.EMAIL
+              </label>
+
+              <div class="input-group col-lg-12">
+                <input
+                  class="form-control"
+                  name="email"
+                  ng-model="EmailReportCtrl.emailReport.email"
+                  autocomplete="off"
+                  type="email"
+                  required/>
+                <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.email.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                  <div ng-messages-include="modules/templates/messages.tmpl.html">
+                  </div>
+                </div>
+              </div>
+            </div>
+            <!-- frequency -->
+
+            <div
+              class="form-group"
+              ng-class="{'has-error' : EmailReportRegistrationForm.frequency.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.FREQUENCY
+              </label>
+
+              <div class="input-group col-lg-12">
+
+                <select
+                  required=""
+                  name="frequency"
+                  ng-model="EmailReportCtrl.emailReport.frequency"
+                  class="form-control">
+                    <option  
+                      ng-repeat="item in EmailReportCtrl.frequencies"
```

Comment:
`ng-repeat` has really bad performance.  In general, you should avoid it where possible.  You can see some discussion of this a long time ago in https://github.com/IMA-WorldHealth/bhima-1.X/issues/143.

In this case, you can use `ng-options` on the `<select>` element.  It would look like this:
```html
<select
  name="frequency"
  ng-model="EmailReportCtrl.emailReport.frequency"
  ng-options="item as item in EmailReportCtrl.frequencies"
  class="form-control"
  required>
</select>
```
It is also best practice to order all options in alphabetical order.  You can do this pretty easily by just adding an `| orderBy: item`.